### ISSUE TITLE
Which product is this package supporting?

### ISSUE THREAD

**Issue by dilab** [dioscouri/omnipay-cybersource#23]
CyberSource has a wide range of integration methods.
It is not clear which method this package is supporting. 

List of CyberSource integration methods:
+ Simple Post
+ Secure Web/Mobile
+ Simple Order
+ SOAP Toolkit
+ Batch Upload

**Comment by eileenmcnaughton** [comment]
I feel like it's simple post - it is the type generally known as a transparent redirect

**Comment by dilab** [comment]
okay, thanks for that. 

I modified the code a bit and made a package that supports Secure Web/Mobile.

https://github.com/dilab/omnipay-cybersource

**Comment by eileenmcnaughton** [comment]
Looks like you didn't start from a fork? I tried to see what a pull request would look like but it didn't seem to want to do one

**Comment by dilab** [comment]
@eileenmcnaughton no, I didn't.  as I did not think you could merge it. So I made it stand-alone. What do you suggest?

**Comment by WillZWL** [comment]
@dilab I also want this package can support Secure Web/Mobile. There need to modified the code to support, I also find there are some bugs need to fix first.

**Comment by WillZWL** [comment]
@eileenmcnaughton I would like to join this project to help this package to support  Secure Web/Mobile

**Comment by rdiaztushman** [comment]
@dilab and @WillZWL You have both been added as contributors to this repo. Thanks!

**Comment by dilab** [comment]
@eileenmcnaughton one of doing that is to rename the current Gateway to indicate which product it supports. My modified package is already supporting Secure Web/Mobile and I do not mind merging it over.

**Comment by eileenmcnaughton** [comment]
Take a look at how Authorize.net supports multiple versions within the same gateway

**Comment by dilab** [comment]
@eileenmcnaughton that is what I mean. they have prefix for each product.

**Comment by eileenmcnaughton** [comment]
Can we grandfather it in?

**Comment by dilab** [comment]
@eileenmcnaughton meaning by default Gateway will be the current one?

**Comment by eileenmcnaughton** [comment]
I would probably suggest keeping Gateway / the top level name as the current product, but grandfathering in a new name for the new product & deprecating the current name of just 'Cybersource' ie. be backward compatible but recognise that down the track there is no top level product

**Comment by adriangordon1231** [comment]
Is it simple post or simple order ?

**Comment by dilab** [comment]
@adriangordon1231 it is been a while since last push. I am not sure if the products support are up to the date.
